FluentConstraints/FluentConstraintSet.swift

```diff
@@ -17,36 +17,36 @@ public class FluentConstraintSet {
         self.firstView = view
     }
 
-    public func build() -> [NSLayoutConstraint] {
+    open func build() -> [NSLayoutConstraint] {
         return constraints.map { $0.build() }
     }
 
-    public func activate() -> [NSLayoutConstraint] {
+    open func activate() -> [NSLayoutConstraint] {
         let constraints = build()
-        NSLayoutConstraint.activateConstraints(constraints)
+        NSLayoutConstraint.activate(constraints)
         return constraints
     }
 
     // MARK: relationship to view
 
-    public var inSuperview: FluentConstraintSet {
+    open var inSuperview: FluentConstraintSet {
         precondition(self.firstView.superview != nil, "View does not have a superview")
         self.constraints.forEach { $0.secondView = self.firstView.superview! }
         return self
     }
 
-    public func onView(view: UIView) -> FluentConstraintSet {
+    open func onView(_ view: UIView) -> FluentConstraintSet {
         self.constraints.forEach { $0.secondView = view }
         return self
     }
 
-    public func asView(view: UIView) -> FluentConstraintSet {
+    open func asView(_ view: UIView) -> FluentConstraintSet {
```

Comment:
Looks like you can use backticks to escape the definition, and then call sites can use it straight up. (At least, that's how it appears when I did something similar in a playground.)

Point taken about the reserved word, though. Apple's advice in the fix-it is to use backticks only if the name is unavoidable. I mean, it's _always_ avoidable...